### PR TITLE
Custom email templates: add function explanation and examples

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Updates the concept content of the functions section to include examples.
- **Is this PR related to a Monolith release?** No.

### Resolves:

* [OKTA-797547](https://oktainc.atlassian.net/browse/OKTA-797547)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/b4a4c4733ddc0ab5f7a529e3e418d819f6b014e6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e1b64bf8-059e-48d3-aa7f-55fb1136db4b) | 65  | 90  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.e12033d9-b749-493d-8da9-5456127c131a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/749b005b767c6835dd17788974b11caed02e06aa/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8cbbdc46-e589-486f-9f66-24ef4b0fd598) | 65  | 90  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.37261f27-0d05-408f-ab77-305456a70a97?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/8e82a87c0e9b8d78393ba38d220a63221d44935c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e7a5ba28-763f-4c8c-9ea9-d8bcc14cbab0) | 66  | 90  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.4d54e426-598c-4f9c-941b-ed285ff1e752?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 1]
```diff
@@ -365,25 +365,44 @@ You can reference any Okta User Profile attribute in your email templates.
...
+
+* Replace function: This example replaces the word `User` with `John` resulting in `"Hello, John!"`:
+
+  `${replace("Hello, User!", "User", "John")}`
+
+* Substring after function: This example returns the substring after the `@` character, resulting in `example.com`:
+
+  `${substringAfter("user@example.com", "@")}`
+
+* To lower case function: This example converts the string `HELLO WORLD` to all lowercase, resulting in `helloe world`:
```
Comment (justinabrokwah-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:386]: nit: typo in `hello world` at the end of the line

**Review by justinabrokwah-okta — COMMENTED** [review]
Looks good overall, just left a nit!

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/94161506577cab92eac81cb863c1d99d8d2b0c8c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a38839b8-17f2-41e3-923e-95d8eb654cc0) | 66  | 90  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.e4b479e7-3a18-4363-808e-0d0b434ee6fd?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by justinabrokwah-okta — APPROVED** [review]

**Pull request merged**